ceph.spec.in

```diff
@@ -839,7 +839,7 @@ rm -rf %{buildroot}
 %{_sbindir}/ceph-create-keys
 %{_sbindir}/rcceph
 %dir %{_libexecdir}/ceph
-%{_libexecdir}/ceph/ceph_common.sh
+%{_libdir}/ceph/ceph_common.sh
```

Comment:
@johnlinp i think @smithfarm mistook us that we wanted to use `/usr/libexec` when packaging rpm. but we don't. we just want to improve the cmake script. and the net result of rpm/deb packaging will be the same.